unlock-app/src/components/interface/locks/CheckoutUrl/index.tsx

```diff
@@ -31,11 +33,46 @@ export const CheckoutUrlPage = () => {
     skipRecipient: true,
   })
 
-  const onAddLocks = (locks: any) => {
+  const handleRecurring = (locks: Record<string, any>) => {
```

Comment:
I don't think this is the rigth place to handle this, we should do it on the specific lock's form

Reply:
If we handle the recurring on the lock form, the default lock will be skipped because it is not handled from the lock form. this also will avoid code duplication (1 to handle default + 1 on lock form)